docusaurus/docs/publish-a-plugin/build-automation.md

```diff
@@ -18,42 +18,171 @@ keywords:
 
 By creating a release tag, the whole process becomes automated, resulting in a zip file that you can submit for publication to the [Grafana plugin catalog](https://grafana.com/plugins)
 
-The zip file can be downloaded from the summary page of the CI workflow.
+You can use the links to the archive and zip files from the release page to make your plugin submission.
+
+## Setup the release workflow
+
+Ensure your repository contains a `.github/workflows/release.yml` file with the following contents: 
+
+```yaml title=".github/workflows/release.yml"
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*' # Run workflow on version tags, e.g. v1.0.0.
+
+jobs:
+  release:
+    permissions:
+      id-token: write
+      contents: write
+      attestations: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: grafana/plugin-actions/build-plugin@main
+        with:
+          # see https://grafana.com/developers/plugin-tools/publish-a-plugin/sign-a-plugin#generate-an-access-policy-token to generate it
+          # save the value in your repository secrets
+          policy_token: ${{ secrets.GRAFANA_ACCESS_POLICY_TOKEN }}
+          # creates a signed build provenance attestation to verify the authenticity of the plugin build
+          attestation: true
+```
+
+:::note
+It is likely this file already exists in your repository. Verify its content and make any necessary changes to match the example.
+:::
+
+## How to trigger the release workflow
+
+To trigger the release workflow you need to push a tag with the format `vX.X.X` to the repository. Typically all of your changes will be merged into `main`, and the tag is applied to `main`
+
+### Create a `vX.X.X` tag with your package manager (Recommended)
+
+The easiest way to create a version tag is using your package manager.
+
+In the following examples a patch version (following [Semantic Versioning](https://semver.org/)) is created:
+
+with [npm](https://docs.npmjs.com/cli/v7/commands/npm-init):
+
+```
+npm version patch
+```
 
-## Create a release tag
+with [yarn](https://yarnpkg.com/lang/en/docs/cli/version/):
+
+```
+yarn version patch
+```
+
+with [pnpm](https://pnpm.io/):
+
+```
+pnpm version patch
+```
+
+This updates your version in the `package.json` file and creates a new Git tag with the format `vX.X.X`. You can change `patch` to `minor` or `major` if you want to create a new minor or major version.
+
+After creating the tag, push it to the repository:
+
+```bash
+git push origin main --tags
+```
 
-A tag with the format `vX.X.X` is used to trigger the release workflow. Typically all of your changes will be merged into `main`, and the tag is applied to `main`
+<details>
+  <summary>Alternatively, create a `vX.X.X` tag manually</summary>
```

Comment:
nit: do we need these instructions? There's missing instructions here to change the version in package.json. The package manager cli commands make this much easier to keep in sync.

Reply:
I don't think we need them and that's why I put them under details. I'll remove them